debchange, debuild, dget, quilt: add pages (#4022)
tldr-pages/tldr: pages/common/quilt.md

# quilt

> > Tool to manage series of patches.

Reviewer: Perhaps this will read better?
```suggestion
> Tool to manage a series of patches.
```

> More information: <https://savannah.nongnu.org/projects/quilt>.

- Import an existing patch from a file:

`quilt import {{path/to/filename.patch}}`

- Create a new patch:

`quilt new {{filename.patch}}`

- Add a file to the current patch:

`quilt add {{path/to/file}}`

> - After you have edited the file, refresh the current patch with your changes:

Reviewer 2: ```suggestion
- After editing the file, refresh the current patch with the changes:
```

`quilt refresh`

> - Apply all patches in the series file:

Reviewer: Perhaps this will read better?
```suggestion
- Apply all the patches in the series file:
```
